doc/dev/mgmt/tests.md

```diff
@@ -15,178 +14,129 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
 
 # Overview
 
-This page is to help you write tests for Azure Python SDK when these tests require Azure HTTP requests.
-The Azure SDK test framework uses the [`azure-devtools`](https://pypi.python.org/pypi/azure-devtools) package,
-which in turn rests upon on a HTTP recording system ([vcrpy](https://pypi.python.org/pypi/vcrpy))
-that enables tests depending on network interaction
-to be run offline.
+This page is to help you write tests for Azure Python SDK when these tests require Azure HTTP requests. The Azure SDK test framework uses the `azure-devtools` package, which in turn rests upon on a HTTP recording system ([vcrpy](https://pypi.python.org/pypi/vcrpy)) that enables tests dependent on network interaction to be run offline.
 
 In this document, we will describe:
--   How to run the tests offline, using previously recorded HTTP interactions,
-    or online, by authenticating with Azure to record new HTTP interactions
+-   How to run the tests offline (using previously recorded HTTP interactions)
+-   How to run the tests online (by authenticating with Azure to record new HTTP interactions)
 -   How to write new tests using our utility classes
 
 # Getting the tests to run
 
-This section describes how to run the SDK tests,
-by installing dependencies into a virtual environment
-and using a helper script and the [pytest](https://docs.pytest.org/en/latest/) test runner
-to choose which tests to run.
+This section describes how to run the SDK tests, by installing dependencies into a virtual environment and using a helper script and the [pytest](https://docs.pytest.org/en/latest/) test runner to choose which tests to run.
 
 ## Running the tests
 
-Azure SDK tests use [pytest](https://docs.pytest.org/en/latest/) test runner.
-To run all the tests, you can use the following command:
+Azure SDK tests use [pytest](https://docs.pytest.org/en/latest/) test runner. To run all the tests, you can use the following command:
```

Comment:
I think it might be better to frame this section as "how to run the tests for one sdk". So, you can be like "in this section, we're running the tests in `azure-mgmt-storage` as an example". Idt the majority of users will be running all of the SDK tests at once